package.json

```diff
@@ -1,60 +1,80 @@
 {
-  "publisher": "vs-code-ai-extension",
-  "name": "vs-code-ai-extension",
-  "displayName": "vs-code-ai-extension",
+  "publisher": "BeanLab",
+  "name": "vscode-byolad",
+  "displayName": "byoLAD",
   "description": "",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "engines": {
     "vscode": "^1.81.0"
   },
+  "license": "SEE LICENSE IN LICENSE",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/beanlab/vs-code-ai-extension.git"
```

Comment:
I had to add the repository to `package.json` to package the extension. I had the url as "https://github.com/beanlab/vs-code-ai-extension" (without the git). Is there a preferred format?

Reply:
I'm not sure if it matters either way, but the docs [example ](https://code.visualstudio.com/api/references/extension-manifest#example)uses the ".git" extension